src/components/book-card/z-book-card/index.stories.ts

```diff
@@ -90,6 +90,43 @@ export const Card = {
     ></z-book-card>`,
 } satisfies Story;
 
+export const WithcoverOverlaySlot = {
```

Comment:
Mettiamola in fondo a tutto, nascondiamola!
E poi ci vuole la C maiuscola, altrimenti viene scritto tutto attaccato, signor Precisini lei oggi ha sonno e non è attento!